rkt/images.go

```diff
@@ -104,6 +105,8 @@ func (f *finder) findImage(img string, asc string, discover bool) (*types.Hash,
 	return h, nil
 }
 
+var errRetrySignature = errors.New("signature is not yet ready and must be re-downloaded")
```

Comment:
This is too specific to be returned on any 202 from `downloadHTTP`, since it's also used for downloading images for example. I think for now just make it `errStatusAccepted`